libs/common-services/src/main/java/uk/gov/di/ipv/core/library/persistence/item/IpvSessionItem.java

```diff
@@ -36,7 +36,7 @@ public class IpvSessionItem implements DynamodbItem {
     private String emailAddress;
     // Only for passing the featureSet to the external API lambdas at the end of the user journey.
     // Not for general use.
-    private String featureSet;
+    private List<String> featureSet;
```

Comment:
I assume we won't be using any feature sets in prod? This could potentially break active sessions that are using feature sets

Reply:
Done-reverted